app/src/main/java/com/example/learingrealmandretrofit/fragment/DeckFragment.kt

```diff
@@ -123,4 +103,19 @@ class DeckFragment : Fragment() {
             bundleOf(DialogCreateOrChangeDeck.deckRealmKey to deckRealm)
         )
     }
+
+    private fun deleteAllDecksRealm() {
+        val config = ConfigRealm.config
+        val realm = Realm.getInstance(config)
+        realm.executeTransactionAsync ({ realmTransaction ->
+            val result = realmTransaction
+                .where(DeckRealm::class.java)
+                .findAll()
+            result.deleteAllFromRealm()
+        }, {
+            realm.close()
+        }, {
+            realm.close()
+        })
+    }
 }
```

Comment:
Configure Android Studio to insert empty line at the end of the file upon saving